clusterloader2/pkg/measurement/util/phase_latency.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"k8s.io/klog"
+)
+
+// Transition describe transition between two phases.
+type Transition struct {
+	From      string
+	To        string
+	Threshold time.Duration
```

Comment:
Why do we need threshold here?

Reply:
We want to display threshold for some metrics - especially when they are slos.

Reply:
OK - so it's for printing only. That's fine